scripts/prometheus.yml

```diff
@@ -6,10 +6,9 @@ scrape_configs:
     static_configs:
       - targets:
           [
-            "172.20.0.2:2379",
-            "172.20.0.3:2379",
-            "172.20.0.4:2379",
-            "172.20.0.5:2379",
+            "node1:9100",
```

Comment:
Where do we define these names? on the test servers?

Reply:
Containers on the same network can access each other by using the other container's name.